packages/core/strapi/lib/services/cron.js

```diff
@@ -1,13 +1,18 @@
 'use strict';
 
 const { Job } = require('node-schedule');
-const { isFunction } = require('lodash/fp');
+const { isFunction, forEach } = require('lodash/fp');
 
 const createCronService = () => {
-  let jobsSpecs = [];
+  let jobsSpecsMap = {};
 
   return {
-    add(tasks = {}) {
+    add(tasks = {}, namespace) {
```

Comment:
I guess this requires to update the documentation? Also: without a default value for `namespace` that would be breaking change, no? Should we add a default value?

Reply:
It is not a documented feature. So it's alright no?

Reply:
`namespace = common` for instance?